src/pages/Content/index.js

```diff
@@ -10,10 +10,18 @@ chrome.runtime.onMessage.addListener(function (msgObj) {
       console.log('Should start recording here');
       sendMessage({ action: 'test', value: 'Hi from content (recorder)!' });
       break;
+    case 'stopRecording':
+      console.log('Should stop recording here');
+      sendMessage({ action: 'test', value: 'Hi from content (recorder)!' });
+      break;
     case 'startPlaying':
       console.log('Should start playing recording...');
       sendMessage({ action: 'test', value: 'Hi from content (player)!' });
       break;
+    case 'stopPlaying':
+      console.log('Should stop playing recording...');
+      sendMessage({ action: 'test', value: 'Hi from content (player)!' });
```

Comment:
Again, the message values and action are not consistent with this event.
Plus here we need to send back Popup an indication if the play finished successfully or not.

Reply:
Same as the other